sickbeard/tvcache.py

```diff
@@ -308,24 +342,16 @@ def shouldClearCache(self):
 
         return False
 
-    def _addCacheEntry(self, name, url, seeders, leechers, size, pubdate, hash, parse_result=None, indexer_id=0):
-
-        # check if we passed in a parsed result or should we try and create one
-        if not parse_result:
+    def _addCacheEntry(self, name, url, seeders, leechers, size, pubdate, hash):
 
-            # create showObj from indexer_id if available
-            showObj = None
-            if indexer_id:
-                showObj = Show.find(sickbeard.showList, indexer_id)
-
-            try:
-                parse_result = NameParser(showObj=showObj).parse(name)
-            except (InvalidNameException, InvalidShowException) as error:
-                logger.log(u"{}".format(error), logger.DEBUG)
-                return None
+        try:
+            parse_result = NameParser().parse(name)
```

Comment:
this fixed an issue from @duramato about not parsing correctly a show with scene exception